lib/engines/content_object_store/app/views/content_object_store/content_block/editions/_schemas.html.erb

```diff
@@ -1,5 +1,23 @@
-<ul class="govuk-list">
-  <% schemas.each do |schema| %>
-    <li><%= link_to_new_block_type(schema) %></li>
-  <% end %>
-</ul>
+<% if @error_message %>
+  <%= render ContentObjectStore::ContentBlockEdition::New::ErrorSummaryComponent.new(
+    error_message: @error_message,
+  ) %>
+<% end %>
+
+<%= form_for :block, url: content_object_store.new_content_object_store_content_block_edition_path, method: :get do |f| %>
```

Comment:
I think Rails is pushing us to start using `form_with` [these days](https://guides.rubyonrails.org/form_helpers.html#using-form-tag-and-form-for) if it's possible.

Reply:
Done! 👍